docs/apidocs/modules.rst

```diff
@@ -0,0 +1,7 @@
+nvflare
+=======
+
+.. toctree::
+   :maxdepth: 4
+
+   nvflare
```

Comment:
do we need this? @nvkevlu @SYangster

Reply:
This change isn't needed we should remove this. The apidocs are automatically generated when building the documentation.

Reply:
@falibabaei can you undo this one as well? Thanks

Reply:
Done